py/server/deephaven/table.py

```diff
@@ -2554,6 +2570,11 @@ def is_refreshing(self) -> bool:
         """Whether this proxy represents a refreshing partitioned table."""
         return self.target.is_refreshing
 
+    @property
+    def update_graph(self) -> _JUpdateGraph:
```

Comment:
Two problems:
1) The API is exposing Java objects to the user.
2) The type hint is referring to a private variable.